packages/atlas/src/components/_buttons/Button/Button.tsx

```diff
@@ -30,6 +30,7 @@ export const Button = React.forwardRef<HTMLButtonElement, ButtonProps>(
         textOnly={textOnly}
         variant={variant}
         iconOnly={iconOnly}
+        data-badge={badge}
```

Comment:
we should add `data-badge` as an emotion prop in this component

Reply:
possibly even make it a string and use `formatNumberShort` so it doesn't break with big numbers

Reply:
Okay, changed that back to allow number as well, it's up to the consumer, doesn't make a difference for the component